### DISCUSSION TITLE
[ Github Package ] This button doesn't work so I am unable to proceed

### DISCUSSION THREAD

**Discussion by gm0ta6** [community/community#50511] [GitHub Education] [answered]
![image](https://user-images.githubusercontent.com/105571933/226399436-0f51d16c-f8d0-4a9e-a1e9-a0928008b0c7.png)
Hey, I'm trying to see which perks I can receive as a student since my teacher told me we are eligible, so can you provide some help cause I'm already approved as a student by github but when I click the button it gets me back to the page where I came from (https://education.github.com/benefits?type=student)

Help please, and thanks!

**Comment by meppu** [comment, marked as answer]
The following path must show all benefits: https://education.github.com/pack#offers

**Reply by meppu** [reply]
You can also visit it from your main page (https://education.github.com/) which must redirect you here in this case (https://education.github.com/globalcampus/student):
![image](https://user-images.githubusercontent.com/112294217/226418296-4c56d85c-14e6-4f77-97be-fce675797a5a.png)
